assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
```

Comment:
This is how it looks now.

![banner_currently_at_977px](https://cloud.githubusercontent.com/assets/15999736/13351802/2c093e24-dc81-11e5-9b86-a8bada97bc0c.png)